src/akvo_unified_log/migrations.clj

```diff
@@ -0,0 +1,59 @@
+(ns akvo-unified-log.migrations
+  (:require [clojure.java.jdbc :as jdbc]
+            [ragtime.repl]
+            [ragtime.jdbc]
+            [taoensso.timbre :as log]))
+
+(defn database-exists? [db-spec db-name]
+  (not (empty? (jdbc/query db-spec
+                           ["SELECT 1 from pg_database WHERE datname=?" db-name]))))
+
+(defn create-event-log-db [db-spec org-id]
+  (log/infof "Creating database %s" org-id)
+  (jdbc/execute! db-spec
+                 [(format "CREATE DATABASE \"%s\"" org-id)]
```

Comment:
I think we want more explicit parameters to the `CREATE DATABASE <db-name> WITH template0 ENCODING 'UTF8'`,

https://github.com/akvo/akvo-provisioning/blob/develop/puppet/modules/unilog/templates/initialize_db.sh.erb#L12-L15